package.json

```diff
@@ -302,7 +302,6 @@
     "@types/vscode-webview": "^1.57.3",
     "@vscode/codicons": "^0.0.33",
     "@vscode/webview-ui-toolkit": "^1.2.2",
-    "axios": "^0.28.1",
```

Comment:
do we need to set `engines` field with this change to require at least Node 18?

Reply:
added